libs/set-algebra/src/Control/Iterate/BaseTypes.hs

```diff
@@ -278,41 +270,6 @@ instance Iter Map.Map where
   isnull = Map.null
   lookup = Map.lookup
 
--- ==========================================================================
--- Basic ViewMap
-
-instance
-  (Monoid coin, Ord coin, Ord cred, Ord ptr, Ord pool) =>
-  Basic (UM.View coin cred pool ptr)
-  where
-  addkv (k, v) m comb = UM.insertWith' comb k v m
-  addpair = UM.insert'
-  removekey = UM.delete'
-  domain = UM.domain
-  range = UM.range
-
-instance
-  (Ord coin, Ord cred, Ord ptr) =>
-  Iter (UM.View coin cred pool ptr)
```

Comment:
Removing instances is a breaking change. Must be reflected in the changelog and in a version bump.